pkg/v1/providers/infrastructure-aws/v1.2.0/yttcc/overlay.yaml

```diff
@@ -66,13 +66,17 @@ spec:
         metadata:
           annotations:
             #! VVV TODO(vui) os-name handling
```

Comment:
Please remove this TODO as we have addressed it.

Reply:
done